Serializer/Serializer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Opstalent\ApiBundle\Serializer;
+
+use \Symfony\Component\Serializer\Serializer as BaseSerializer;
+
+/**
+ * {@inheritDoc}
+ */
+class Serializer extends BaseSerializer
+{
+    /**
+     * @param array $context
+     * @param $data
+     * @return array
+     */
+    protected function buildContext(array $context = [], $data)
+    {
+        if(array_key_exists("top", $context)
+            && is_object($data)
+        ) {
+            unset($context["top"]);
+            $context['entity'] = $data;
```

Comment:
can you explain this logic?

Reply:
it take first object (so it will be collection entity for list) as cotnext